deps/checksums/openblas

```diff
@@ -1,94 +1,94 @@
+OpenBLAS.v0.3.10+8.x86_64-linux-musl-libgfortran3.tar.gz/md5/f60d89103b9a1d5cbf0ffd82d5519176
+OpenBLAS.v0.3.10+8.x86_64-linux-musl-libgfortran3.tar.gz/sha512/7bffc4b63c038a1572d6082dac239bbba0b3e77f13c5688b99010fca7395d64eed214f6873e99466bb191e3e3a5a82a6a9e0d45bdd3785a296f14e9de0dab36a
+OpenBLAS.v0.3.10+8.x86_64-linux-musl-libgfortran4.tar.gz/md5/a7165b675d95ad8f2fdfbad6a81c4234
+OpenBLAS.v0.3.10+8.x86_64-linux-musl-libgfortran4.tar.gz/sha512/f1cfef8009b2b207df568f1d85dc20afcb7391acec7d3fc5802fdb2891498d99f4d2cb43421d415ec0d7ec8d55b899d1d967824a28a29a1f741eb11f29708bd1
+OpenBLAS.v0.3.10+8.x86_64-linux-musl-libgfortran5.tar.gz/md5/b90c241608aa03db5ad620e486987b77
+OpenBLAS.v0.3.10+8.x86_64-linux-musl-libgfortran5.tar.gz/sha512/3c1cb045a04acd57a7d5720fa8fb37ecec4327e8c0e6f040246bf3e9c653130ee9da9ee51fb707a753e1d5bbc60abb740a334fed0c741aeb118dc7a7a3d9735d
+OpenBLAS.v0.3.10+8.x86_64-unknown-freebsd-libgfortran3.tar.gz/md5/f385b35d736c1e80311f795a0814ba04
+OpenBLAS.v0.3.10+8.x86_64-unknown-freebsd-libgfortran3.tar.gz/sha512/54a6870225fa75040113614561fe1c495efc20fa279a5a597f492935420699050f0efe0208a73145f63b3a9722c7687414b0fcb3893be653005f76991e13b416
+OpenBLAS.v0.3.10+8.x86_64-unknown-freebsd-libgfortran4.tar.gz/md5/b8a973b0fa79e5b2a3ec02df83585c64
+OpenBLAS.v0.3.10+8.x86_64-unknown-freebsd-libgfortran4.tar.gz/sha512/8ca5b01de2ca124ee5d9ab5d9dac607ce50e1af8fbeaa15884a38ca2630e06e9cb8b7df236da78114fff8f2a686ebfde05438037df46acaa0c29dc4289f35702
+OpenBLAS.v0.3.10+8.x86_64-unknown-freebsd-libgfortran5.tar.gz/md5/8d9ef0b998689c0070d7932b36bada20
+OpenBLAS.v0.3.10+8.x86_64-unknown-freebsd-libgfortran5.tar.gz/sha512/349739fa51524ddd23fb0219c4249f0f5ad36f42b66baabec4ac7c2dead860b9efa07cc3652374f4ad572ee2e5851ac2ec5556cfa6df0e743c7cc9ba7a6cac44
+OpenBLAS.v0.3.10+8.x86_64-w64-mingw32-libgfortran3.tar.gz/md5/a680bfa877cfc313a7e8b2c87469d615
+OpenBLAS.v0.3.10+8.x86_64-w64-mingw32-libgfortran3.tar.gz/sha512/2fd93985e36d62e8d583d07d32e1d37848f47b46dc1f2278909bad98a05c6c47a0ac29fb6217fe017bb66f520b0ef0bde49af370d372d1d8dd43a5f104bd5f7f
+OpenBLAS.v0.3.10+8.x86_64-w64-mingw32-libgfortran4.tar.gz/md5/d616bef77c9d1b0f199ebcb8a5ccf908
+OpenBLAS.v0.3.10+8.x86_64-w64-mingw32-libgfortran4.tar.gz/sha512/b12a9310be4a934b86b073ed03cea18e30f01039399162409b1dff1e097d75430cbaa1d7d5cf0972d32907137c54d455e266aeb854ad86f7bbb309b01c0d9dd1
+OpenBLAS.v0.3.10+8.x86_64-w64-mingw32-libgfortran5.tar.gz/md5/733f0803a1c835a94beeec70ef713512
+OpenBLAS.v0.3.10+8.x86_64-w64-mingw32-libgfortran5.tar.gz/sha512/97fcbfd8af8ad281203529e2d31bf5a4892831ebf2805d09b373ce3bd5fb108f33436ea42b08287d283b7b45e4be0f5433409a9bc5d8252941f30cd03851a839
 openblas-63b03efc2af332c88b86d4fd8079d00f4b439adf.tar.gz/md5/3d692acc6927454f620a4c493bdb159d
 openblas-63b03efc2af332c88b86d4fd8079d00f4b439adf.tar.gz/sha512/cf89f6db1b6366833d29a1dc718ea0b8f61d162f70695c33fc94afbaba232605630a7a7cc3d3b9bed7493ec85402b65180ca99c3101de7141d6f2919318f55c1
```

Comment:
The above two lines of checksums are not updated, is this the expected behavior of the `contrib/refresh_checksums.mk` script?

Reply:
Yes those are the sources